governance.md

```diff
@@ -0,0 +1,38 @@
+# OceanGliders format goverance proposal
+
+## Status
+
+2022-06-27 draft policy based on the existing readme for review and feeedabck
+from the OGDMTT and OG steering team
+
+## Data format
+
+
+### Who is reviewing the issues?
+A sub-group of the OceanGliders data management team will be in charge of reviewing the issues.
+
+### How to present an issue?
+It is requested to provide a detailed explanation of the issue with an example. Suggestions can be added to the issue.
+
+### How do we proceed with an issue?
+The format will need to grow and evolve with time while being managed effectively by the OGDMTT
+
+1. The team will analyze the issue formally. If the issue is not detailed enough or if there is no example associated with the issue, the issue will be rejected. If the issue is accepted for review, then the team will study it and provide an answer. The issue can lead to a modification of the format or not. If there is no consensus on the solution, the suggested solution will be rejected and the format will remain unchanged. However, a new branch can be created to demonstrate the benefit of the solution. The same issue can be submitted again with a demo.
+
+2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for approval. Stakeholers will be noftifed of the change at this point.
+
+## Vocabularies
+The vocabulary collections are more dynamic than the format. When terms are added to vocabulary collections there will be no need to update the format with the vocabulary contected being governed by a dedicate GitHub page for the collection.
```

Comment:
Need to develop the introduction to make it accessible for non-experts

Reply:
Vocabularies tend to be managed via the hosting vocabulary services and governance teams

Reply:
Section on adding new terms to existing vocabularies

Reply:
Add in that people need to work collaboratively with the OGDMTT